### ISSUE TITLE
UI Loading on every spawn

### ISSUE THREAD

**Issue by rlrpuk1** [PainedPsyche/cui_character#15]
UI is loading on every spawn

Followed all instructions. Using extendedmode.

**Comment by ArmoredSoldiers** [comment]
Same

**Comment by kobel240** [comment]
indeed, I have this same error. There's already a reply on this (https://forum.cfx.re/t/release-cleanui-cui-character/1914841/183?u) scroll a little further and you'll find some solutions, however non of these have worked for me. hmu if you figured it out.
